nemo/collections/nlp/parts/nlp_overrides.py

```diff
@@ -174,6 +176,7 @@ def __init__(self) -> None:
             logging.warning("Apex was not found. Using model parallel or megatron models will error out.")
 
     def save_to(self, model, save_path: str):
+        # TODO: make save_to work with always_save_nemo for model parallel
```

Comment:
Is `save_to` works with `always_save_nemo` ?

Reply:
Yeah I think @ericharper fixed this. Will remove the comment.